pycoast/cw_base.py

```diff
@@ -924,24 +947,131 @@ def add_cities(self, image, area_def, citylist, font_file, font_size,
                 except ValueError as exc:
                     logger.debug("Point not added (%s)", str(exc))
                 else:
-
                     # add_dot
                     if ptsize is not None:
                         dot_box = [x - ptsize, y - ptsize,
                                    x + ptsize, y + ptsize]
                         self._draw_ellipse(
                             draw, dot_box, fill=outline, outline=outline)
-                        text_position = [x + 9, y - 5]  # FIX ME
+                        text_position = [x + 9, y - 5]  # FIXME
                     else:
                         text_position = [x, y]
 
                 # add text_box
-                    self._draw_text_box(draw, text_position, city_name, font,
-                                        outline, box_outline, box_opacity)
+                    self._draw_text_box(draw, text_position, city_name, font, outline,
+                                        box_outline, box_opacity)
                     logger.info("%s added", str(city_name))
 
         self._finalize(draw)
 
+    def add_points(self, image, area_def, points_list, font_file, font_size=12,
+                   symbol='circle', ptsize=6, outline='black', fill='white', **kwargs):
+        """Add a symbol and/or text at the point(s) of interest to a PIL image object.
+        :Parameters:
```

Comment:
If you generate the sphinx docs, does this complain? I'm wondering if there needs to be a blank line above Parameters.